packages/sass-node-modules-importer/source/logger.ts

```diff
@@ -0,0 +1,10 @@
+import { Options } from './types'
+
+export const log = (
+	options: Options,
+	message: string,
+	...more: unknown[]
```

Comment:
don't know if it's safe to allow the log usage to accept any other args 
destructing the more in the console.debug call would be faulty in the case of a boolean for example

Reply:
This is handled by `console.debug` accepting `any[]`